public/components/incontext_insight/index.scss

```diff
@@ -0,0 +1,144 @@
+// TODO: fix the styling due to not wanting the chat icon to push elements
+// Might be worth reconsidering the icon for absolute positioning
+.hoverEffectUnderline {
+  border: solid 2px transparent !important;
+  margin: -2px -5px -5px -7px !important;
+}
+
+.hoverEffect0 {
```

Comment:
I am afraid this class name may be too generic, can we add a prefix like insight-context?

Reply:
good call
added.